tests/test.spec.js

```diff
@@ -805,4 +807,40 @@ describe('bnid', () => {
       }
     });
   });
+  describe('secret key seed', () => {
+    it('should generate a secret key seed', async () => {
+      let secretKeySeed;
+      let err;
+      try {
+        secretKeySeed = await generateSecretKeySeed();
+      } catch(e) {
+        err = e;
+      }
+      should.exist(secretKeySeed);
+      should.not.exist(err);
+      secretKeySeed.should.be.a('string');
+      secretKeySeed.length.should.equal(47);
```

Comment:
Is this always 47?

Reply:
For the current test case when using the default params, it is always 47